### DISCUSSION TITLE
How do I know when Neovide is on?

### DISCUSSION THREAD

**Discussion by enbugger** [neovide/neovide#943] [Q&A] [answered]
I was looking for something like below but didn't find anything in the docs:
```vim
" .vimrc
"  This does not work
if has('neovide')
    let g:neovide_cursor_vfx_mode = "sonicboom"
endif
```
What is the correct way to do that if any?

**Comment by Kethku** [comment, marked as answer]
g:neovide is defined on startup

https://github.com/neovide/neovide/discussions/609
https://github.com/neovide/neovide/discussions/919
https://github.com/neovide/neovide/issues/835
https://github.com/neovide/neovide/issues/229
https://github.com/neovide/neovide/issues/24
;)

**Reply by enbugger** [reply]
thanks :)
